hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/overview/overview.tsx

```diff
@@ -227,6 +231,9 @@ export class Overview extends React.Component<Record<string, object>, IOverviewS
           <Col xs={24} sm={18} md={12} lg={12} xl={6}>
             <OverviewCard loading={loading} title='Keys' data={keys.toString()} icon='file-text'/>
           </Col>
+          <Col xs={24} sm={18} md={12} lg={12} xl={6}>
+            <OverviewCard loading={loading} title='Pending Key Deletions' data={keysAwaitingDeletion !== undefined && keysAwaitingDeletion.toString()} icon='delete'/>
```

Comment:
is this `keysAwaitingDeletion !== undefined` check necessary? the above cards don't have this check either. the backend changes were done in #4406 by @ArafatKhan2198, what do you think? I don't see anything that would require the check.

Reply:
@dombizita @ArafatKhan2198 

Done changes suggested by Zita removed undefined condition in latest commit.